src/diffusers/models/unets/unet_motion_model.py

```diff
@@ -394,7 +394,7 @@ def from_unet2d(
         has_motion_adapter = motion_adapter is not None
 
         # based on https://github.com/guoyww/AnimateDiff/blob/895f3220c06318ea0760131ec70408b466c49333/animatediff/models/unet.py#L459
-        config = unet.config
+        config = dict(unet.config)
```

Comment:
@DN6 
currently, we will modify the original 2d unet's config - even though we do not use it here, we create a new unet motion model instead